src/commands/database-remove.js

```diff
@@ -38,32 +40,199 @@ module.exports = new Command("database:remove <path>")
           ". Are you sure?",
       },
     ]).then(function() {
+      if (!options.concurrent) {
```

Comment:
More common/concise expression of this would be:

    options.concurrent = options.concurrent || 100;